docs/development/l10n.rst

```diff
@@ -159,15 +159,22 @@ HTML content
 ^^^^^^^^^^^^
 
 Some source strings represent HTML that will be presented in the
-SecureDrop web interface. It can be hard to tell what to translate,
-since HTML is source code and changing the wrong thing can break the
-page layout.
+SecureDrop web interface.
 
-One thing you should always translate are ``alt`` attributes of image
-elements. Image elements (``<img>``) in HTML place a picture on the
+HTML elements (embraced by in ``<``, ``>``, example: ``<strong>``)
+can contain multiple so-called *attributes*.
+
+The text of the two attributes called ``alt`` and ``title``
+should be translated. The text of the other attributes should not
+be translated.
+
+Attribute ``alt``
+"""""""""""""""""
+
+Image elements (``<img>``) in HTML place a picture on the
 page. People with visual impairments rely on a special note on the
```

Comment:
I would write "Because people with visual impairments rely ..., it is necessary ...".

Right now you say "People with visual impairments rely ..., it is necessary ..." which sounds a bit odd in English because the two clauses of the sentence are not connected.